### PR TITLE
fix: native ads

### PR DESCRIPTION
**Pull request by parkjh**

### CODE REVIEW

**Comment by parkjh** [comment]
all article ssr tests are failing as it cannot find component with id `header`...

but after further debugging see test videos https://drive.google.com/open?id=1ol0nVR_qvQrlUT6lEOddm3C9O1j-6Dlc it seems that the pages are rendered but I wonder if there was any work done recently that changed the article-header component?

File: packages/image/src/modal-image/modal-image.js [hunk 1]
```diff
@@ -130,7 +131,11 @@ class ModalImage extends Component {
           </View>
         </Modal>
         <Button onPress={this.showModal}>
-          <Image {...this.props} onLayout={this.onLowResLayout} />
+          <Image
+            {...this.props}
+            lowResSize={lowResSize}
+            onLayout={this.onLowResLayout}
```
Comment (ValGeorgiev) [inline packages/image/src/modal-image/modal-image.js]: this shouldn't be in this PR, could we clear it, please?

File: packages/ad/__tests__/utils/ad-init-apstag.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+export default () => {
+  let mock;
+  let initOptions;
+
+  beforeEach(() => {
+    ({ mock, initOptions } = makeAdInitMocks());
+  });
+
+  it("do not sets up Amazon bidding if amazon account ID is not set", () => {
```
Comment (aronshamash) [inline packages/ad/__tests__/utils/ad-init-apstag.js]: `does not set up`

File: packages/ad/__tests__/utils/ad-init-apstag.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  it("get Amazon Config with a commercial section", () => {
+    const init = adInit(initOptions);
+    expect(
+      init.apstag.getConfig({
+        slots: [],
+        networkId: "3048",
+        adUnit: "d.thetimes.co.uk",
+        section: "news"
+      })
+    ).toEqual([]);
```
Comment (aronshamash) [inline packages/ad/__tests__/utils/ad-init-apstag.js]: Sorry don't understand this test. are you expecting the config to be empty ?

Comment (parkjh) [reply packages/ad/__tests__/utils/ad-init-apstag.js]: @aronshamash yes

Comment (aronshamash) [reply packages/ad/__tests__/utils/ad-init-apstag.js]: @parkjh shouldn't the test desc reflect you expect empty config ?

**Pull request merged**